From 61878fe0ea004da2fe176d49845f24484ed5973c Mon Sep 17 00:00:00 2001
From: Tomas Zak <tomas.zak@turris.com>
Date: Wed, 23 Oct 2024 19:57:27 +0200
Subject: [PATCH] uboot-mpc85xx: change board size for turris1x

Change board size for turris1x due to small size for uboot
---
 .../uboot-mpc85xx/patches/change_board_size.patch     | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 package/boot/uboot-mpc85xx/patches/change_board_size.patch

diff --git a/package/boot/uboot-mpc85xx/patches/change_board_size.patch b/package/boot/uboot-mpc85xx/patches/change_board_size.patch
new file mode 100644
index 0000000000..daa2c5246a
--- /dev/null
+++ b/package/boot/uboot-mpc85xx/patches/change_board_size.patch
@@ -0,0 +1,11 @@
+--- a/configs/turris_1x_sdcard_defconfig
++++ b/configs/turris_1x_sdcard_defconfig
+@@ -20,7 +20,7 @@ CONFIG_PCIE1=y
+ CONFIG_PCIE2=y
+ CONFIG_PCIE3=y
+ CONFIG_HAS_BOARD_SIZE_LIMIT=y
+-CONFIG_BOARD_SIZE_LIMIT=1048576
++CONFIG_BOARD_SIZE_LIMIT=1485760
+ CONFIG_MP=y
+ CONFIG_FIT=y
+ CONFIG_FIT_VERBOSE=y
-- 
2.47.0

